Doc/Tutorial/chapter_advanced.tex

```diff
@@ -223,7 +223,7 @@ \subsection{FreqTable}
 \label{sec:freq_table}
 
 \begin{minted}{python}
-FreqTable.FreqTable(UserDict.UserDict)
+FreqTable.FreqTable(dict)
```

Comment:
This example does not seem to make much sense (before or after the edit).

Reply:
The `FreqTable` class doesn't seem to make much sense.. it doesn't do much.